pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -54,6 +56,26 @@ func GenerateLoggingCredentialsBasicSecret(lc loggedcluster.Interface) *v1.Secre
 	return &secret
 }
 
+func GetLogin(lc loggedcluster.Interface, credentialsSecret *v1.Secret, user string) (string, error) {
+
+	login, ok := credentialsSecret.Data[fmt.Sprintf("%suser", user)]
```

Comment:
What is this `%suser` format ?

Reply:
It's a concatenation of `user` (the variable) and `"user"` (the string), to create composite keys.
Because the secret looks like this:
```
data:
  readpassword: xxxx
  readuser: xxxx
  writepassword: xxxx
  writeuser: xxxx
  wc1password: xxxx
  wc1user: xxxx
```
It seems that secrets don't support complex structures, so if I don't want to encode/decode the whole secret each time I have to use composite keys.